src/routes/choice-exercises/WordChoiceExerciseScreen.tsx

```diff
@@ -15,12 +15,15 @@ interface WordChoiceExerciseScreenPropsType {
 const WordChoiceExerciseScreen = ({ navigation, route }: WordChoiceExerciseScreenPropsType): ReactElement | null => {
   const { extraParams } = route.params
   const { trainingSetId } = extraParams
-  const { data: documents, loading } = useLoadDocuments(trainingSetId)
+  const response = useLoadDocuments(trainingSetId)
+  const [documents, setDocuments] = useState<DocumentType[] | null>(null)
```

Comment:
Is this used anywhere? I think this whole exercise does not work like this. `documents` will always be `null` (initial value) as it is never reset to anything else. Instead you should probably just revert this file.